projects/core/components/error/error.component.ts

```diff
@@ -1,17 +1,21 @@
+import {AsyncPipe, NgIf} from '@angular/common';
 import {
     ChangeDetectionStrategy,
     Component,
     HostListener,
     inject,
     Input,
 } from '@angular/core';
-import {tuiIsString, TuiValidationError} from '@taiga-ui/cdk';
+import {tuiIsString, TuiLetDirective, TuiValidationError} from '@taiga-ui/cdk';
 import {tuiFadeIn, tuiHeightCollapse} from '@taiga-ui/core/animations';
 import {TUI_ANIMATIONS_SPEED, TUI_DEFAULT_ERROR_MESSAGE} from '@taiga-ui/core/tokens';
 import {tuiToAnimationOptions} from '@taiga-ui/core/utils';
+import {PolymorpheusModule} from '@tinkoff/ng-polymorpheus';
 
 @Component({
+    standalone: true,
     selector: 'tui-error',
+    imports: [NgIf, TuiLetDirective, AsyncPipe, PolymorpheusModule],
```

Comment:
maybe to put `ngIf` and `AsyncPipe` firstly ?

Reply:
fixed